lib/phoenix/html/link.ex

```diff
@@ -46,4 +46,40 @@ defmodule Phoenix.HTML.Link do
       end
     end
   end
+
+  @doc """
+  Generates a button that uses a regular HTML form to submit to the given URL.
+  Useful to ensure that links that change data are not triggered by
+  search engines and other spidering software.
+
+  ## Examples
+
+      <%= button("hello", to: "/world") %>
+
+  generates
+
+      <form action="/world" class="button" method="post">
+        <input name="_csrf_token" value=""><input type="submit" value="hello">
+      </form>
+
+  ## Options
+
+    * `:to` - the page to link to. This option is required
+
+    * `:method` - the method to use with the link. Defaults to :post.
+
```

Comment:
Please add a note that all other options are forwarded to the underlying form and link to `Phoenix.HTML.Form.form_tag/2`. :)